modules/define.py

```diff
@@ -1,24 +1,42 @@
 #--require-config wordnik-api-key
 
 import Utils
+import time
 
-URL_WORDNIK = "http://api.wordnik.com:80/v4/word.json/%s/definitions"
+URL_WORDNIK = "https://api.wordnik.com/v4/word.json/%s/definitions"
+URL_WORDNIK_RANDOM = "https://api.wordnik.com/v4/words.json/randomWord"
+
+RANDOM_DELAY_SECONDS = 3
 
 class Module(object):
     def __init__(self, bot, events, exports):
         self.bot = bot
+        self.last_called = 0
+        self.events = events
+
         events.on("received.command.define").hook(self.define,
             help="Define a provided term", usage="<phrase>")
 
+        events.on("received.command.randomword").hook(self.random_word,
+            help="Generate a random word!")
+
+    def get_definition(self, event):
+        word = event["args"] if "args" in event else event
```

Comment:
what you should do here is

```
def get_definition(self, event):
    self._get_definition(event["args"])
def _get_definition(self, word):
    # code here
```

So that you don't have to do any inspection on the argument and two places can reuse the same code